service/updateService.go

```diff
@@ -83,21 +84,24 @@ func (service UpdateServiceImpl) GetUpdates(ctx context.Context, catalogId strin
 		sortOrderInt = 1
 	}
 
-	databaseUpdates, err := service.UpdateRepository.GetUpdates(ctx, query, page, size, sortByCol, sortOrderInt)
+	databaseUpdates, count, err := service.UpdateRepository.GetUpdates(ctx, query, page, size, sortByCol, sortOrderInt)
 	if err != nil {
 		logrus.Error("Get updates failed")
 		logging.LogAndPrintError(err)
 		return model.Updates{}, http.StatusInternalServerError
 	}
 
 	if databaseUpdates == nil {
-		logrus.Error("No updates found")
+		logrus.Error("No concept updates found")
```

Comment:
Selv om det atm kun er snakk om begrep så vil dette utvides til mer, vi kommer feks til å legge in historikken til service-catalog snart.
```suggestion
		logrus.Error("No updates found")
```